content/en/dashboards/sharing.md

```diff
@@ -19,7 +19,39 @@ further_reading:
 
 ## Overview
 
-Shared graphs and screenboards allow you to display metric, trace, and log visualizations outside of Datadog.
+Shared dashboards and graphs allow you to display metric, trace, and log visualizations outside of Datadog.
+
+## Dashboards
+
+**Share an entire dashboard publicly by generating a URL**:
+1. On the dashboard's page, click the settings cog in the upper right.
+2. Choose the **Generate public URL** option.
+3. Choose configuration options under "Time & Variable Setings" for "Allow changing timeframe" and/or template variables visible tags.
+4. Copy the URL and click **Done**.
+
+**Share with individual emails**: 
+1. On the dashboard's page, click the settings cog in the upper right.
+2. Choose the **Generate public URL** option.
+3. Select "Only specified people" can access this dashboard.
+4. Input the emails you would like to share your dashboard with.
+5. Choose configuration options under "Time & Variable Setings" for "Allow changing timeframe" and/or template variables visible tags.
+6. (Optional) You can copy the URL to share, but they will also receive an email with a link.
+7. Click **Done**.
+
+**Note**: 
+- The created URL has live, read-only access to the contents of that specific dashboard.
+- Dashboard [template variable][4] selectors are only present on dashboards when you configure visible tags. The template variables' default values are the default values set in Datadog. 
+- Additionally, widgets based on APM traces queries do not display data on public dashboards. All log based queries will show data, except the Log Stream widget.
```

Comment:
```suggestion

```

Reply:
moved to the top of the section

Reply:
I feel like adding what we don't do to the top of the section doesn't feel great - I'd rather keep it at the bottom

Reply:
ok, reworked to keep the "doesn't do" down here.